coalals/utils/log.py

```diff
@@ -0,0 +1,14 @@
+import logging
+from sys import stdout
+
+
+def configure_logger():
+    logging.basicConfig(stream=stdout, level=logging.INFO)
```

Comment:
I am not sure if we could log to stdout since we will have a mode stdio, while we could keep it. If it does not work, we could change it to stderr later.